cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -317,19 +317,19 @@ spec:
       served: true
       storage: true
       subresources:
-        status: {}
+        status:
 status:
   acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
+    kind:
+    plural:
+  conditions:
+  storedVersions:
```

Comment:
I'm a little concerned about these changes. It seems like the YQ parsing is getting rid of the empty values.

Reply:
Fixed it by using `yq read` to parse each intermediate file into the output rather than parsing the whole output file at once.